docs/api-reference/core/layer.md

```diff
@@ -597,6 +597,13 @@ Create a copy of this layer, optionally change some prop values.
 Arguments:
 - `overrideProps` (Object, optional) - layer props to update.
 
+Remarks:
+- When using React, instead of calling `layer.clone(...)` you should construct a new layer instance on every render. Instead of the above, use the following each time (both initialization and re-rendering):
```

Comment:
I don't see why this is necessary, we have a [whole section on React](https://deck.gl/docs/get-started/using-with-react) with example usage and on this page it is mentioned that: 

```
    Layer methods are designed to support the creation of new layers or layer sub-classing and are NOT intended to be called by applications.
```